lib/ansible/plugins/callback/profile_tasks.py

```diff
@@ -44,6 +44,11 @@
 '''
 
 EXAMPLES = '''
+example: >
```

Comment:
This is a nice addition. I'd expand it by putting a note in line 25 above pointing people to the examples: 

```
requirements:
  - whitelisting in configuration - see examples section below for details
```

Reply:
Done.